core/src/main/java/nl/nn/adapterframework/pipes/IteratingPipe.java

```diff
@@ -414,18 +414,34 @@ public void waitForResults() throws SenderException, IOException {
 				try {
 					guard.waitForAllResources();
 					int count = 0;
+					List<Throwable> exceptions = new ArrayList<>();
 					for (ParallelSenderExecutor pse : executorList) {
 						count++;
-						String itemResult;
+						String itemResult="";
```

Comment:
Why are you assigning this variable? It shouldn't be necessarily?
```suggestion
						String itemResult;
```